content/blog/lift-and-transform-apps-with-hpe-csi-driver-for-kubernetes.md

```diff
@@ -0,0 +1,283 @@
+dept_emp	331603	d95ab9fe07df0865f592574b3b33b9c741d9fd1b
+dept_manager	24	9687a7d6f93ca8847388a42a6d8d93982a841c6c
+employees	300024	4d4aa689914d8fd41db7e45c2168e7dcb9697359
+salaries	2844047	b5a1785c27d75e33a4173aaa22ccf41ebd7d4a9f
+titles	443308	d12d5f746b88f07e69b9e36675b6067abb01b60e
+table_name	records_match	crc_match
+departments	OK	ok
+dept_emp	OK	ok
+dept_manager	OK	ok
+employees	OK	ok
+salaries	OK	ok
+titles	OK	ok
+computation_time
+00:00:49
+summary	result
+CRC	OK
+count	OK
+```
+
+In the remaining tests throughout this blog, only the summary results will be displayed.
+
+# Destination state
+
+When the transition has completed, the Employees database should be running on a Kubernetes cluster using persistent storage provided by the HPE CSI Driver for Kubernetes backed by a HPE Nimble Storage array. The procedures are similar regardless of backend, HPE Alletra and Primera included.
+
+![MariaDB running on Kubernetes](/img/slide2.png "MariaDB running on Kubernetes")
+
+While the destination state is a virtualized Kubernetes cluster within the same vSphere environment, that is not a requirement, it can be any hypervisor, bare-metal or otherwise as long as the destination cluster is compatible with the HPE CSI Driver for Kubernetes and any of the supported backends.
+
+# Initial transition
+
+The first step in become becoming agile with the dataset of interest (the MariaDB database in this case) is to transition the virtual machine’s storage to a VMware vSphere vVol from the legacy NFS datastore. This is done with VMware vSphere Storage vMotion.
+
+![VMware vSphere Storage vMotion example.](/img/vmotion-screen-shot-2022-06-14-at-12.08.27-pm.png "VMware vSphere Storage vMotion example.")
+
+Another important detail is that the data that matters to the application reside in a filesystem compatible with the destination Kubernetes cluster and without partitioning schemes or volume managers. We know for a fact that MariaDB stores all its data on `/var/lib/mysql` so let’s investigate that path.
+
+```markdown
+$ df -h /var/lib/mysql
+Filesystem      Size  Used Avail Use% Mounted on
+/dev/sda         64G  1.2G   63G   2% /var
+```
+
+If this is not the case, migrate the data within the VM to a new blank disk.
+
+Once the disk(s) have been migrated over the HPE Nimble Storage array, each individual virtual disk is represented as a standalone volume along with the VMs metadata volumes.
+
+![vVol Datastore Volumes](/img/vvol-screen-shot-2022-06-14-at-1.12.59-pm.png "vVol Datastore Volumes")
+
+The 64GB volume represents the `/var` filesystem subject for transforming into a Kubernetes persistent volume.
+
+# Iterate and learn
+
+In this next phase, all configuration and testing are performed directly from the Kubernetes cluster. An admin needs to install the HPE CSI Driver for Kubernetes, create a `Secret` that represents the backend user or tenant and setup a `StorageClass` designed to allow importing of volumes.
```

Comment:
Is "phase" the right word? If it is, then spell out what it is a phase of. "In this next phase of the project" or something like that. But maybe it should be "Now, all configuration and testing are..." Or "In this section, I'll show you how all configuration and testing are performed ..."

It's just not quite clear...  Also, please modify the end of this paragraph to read:
"... designed to allow **the** importing of volumes."